src/core/Settings.js

```diff
@@ -863,6 +866,7 @@ function Settings() {
                 bufferTimeAtTopQualityLongForm: 60,
                 initialBufferLevel: NaN,
                 stableBufferTime: 12,
+                hybridSwitchBufferTime: NaN,
```

Comment:
Should this default to 12 given the comment above?

Reply:
Good spot! Forgot to change the default value to be `stableBufferTime` in documentation. Is it clearer with the change?